modules/repository/license.go

```diff
@@ -111,3 +241,82 @@ func getLicensePlaceholder(name string) *licensePlaceholder {
 	}
 	return ret
 }
+
+// updateRepoLicenses will update repository licenses col if license file exists
+func updateRepoLicenses(ctx context.Context, repo *repo_model.Repository, commit *git.Commit) error {
```

Comment:
It's better to move these to service layer.

Reply:
This will have many changes, maybe do it later.

Reply:
But this is a newly added code. Why not?

Reply:
Done.